sunpy/cm/__init__.py

```diff
@@ -1,7 +1,12 @@
-"""
-The following colormaps are provided by this module:
-"""
-from sunpy.cm.cm import *  # noqa
+import warnings as _w
+from sunpy.util.exceptions import SunpyDeprecationWarning
 
-for cmname in cmlist.keys():  # noqa
-    __doc__ += f"\n* '`~sunpy.cm.cm.{cmname}`'\n"
+_w.warn("The sunpy.cm module has been moved in 1.1 to sunpy.visualization.colormaps, it will be removed in 2.1", category=SunpyDeprecationWarning)
```

Comment:
How about "The functionality of the sunpy.cm module is now in sunpy.visualization.colormaps as of SunPy 1.1.  The ability to import sunpy.cm as a module may be removed in a future version of SunPy."